feat: add auto-login doc (#83)
graasp/docs: docs/user/player/advanced/auto-login.md

# Auto Login

:::warning

> This feature requires a good understanding of Graasp and might be for the more technically inclined.

Reviewer: ```suggestion
This feature requires a good understanding of Graasp and might be for users that are more technically inclined.
```

:::

> In this article we will use the term `itemLogin` to designate the possibility to log a user with a pseudonym.

Reviewer: ```suggestion
In this article we will use the term `itemLogin` to designate the possibility to log a user only with a pseudonym.
```
Or should we have a page about "itemlogin" to reference to? I think it's more "login without an email" than "login with a username"

## Purpose {#purpose}

> This feature allows to force the itemLogin account used for a lesson by providing a url that contains all necessary information to the user.

Reviewer: ```suggestion
This feature allows to force which itemLogin account will be used for a lesson by providing a url that contains all necessary information for the user.
```

##  Use cases {#use-cases}

Potential use-cases for this feature include:

> - Studies (like Prolific) where you can provide a personalized link to each participant and you **need** each participant to use a very specific username (for data analysis purposes for example)

Reviewer: "you need" but "maybe for example" for analysis purpose. Do you need it or not? Maybe "you want" fits better.

> ## Setting up your lesson {#setup}

Reviewer: You use "lesson" but your use-case is a "study". Should we be more generic? Capsule? Activity?

In this section we will go over the process for setting up auto login.

1. Create an item

> 2. Go to this item sharing page

Reviewer: ```suggestion
2. Go to this item's sharing page
```

> 3. In the sharing page enable "Pseudonymized" access with "Pseudonym"

Reviewer: Have a warning that it does not work with password?

   ![allow access for pseudonymised users](./2024-10-23-pseudo-access.png)

4. Create the link to share with users

>     1. Go to the item page in the player, it should look something like this:

Reviewer: ```suggestion
    1. Go to the item page in the player, it should look like this:
```
